routers/routes.js

```diff
@@ -1,11 +1,12 @@
 const express = require("express");
-
 const controllers = require("../controllers/controllers");
-
+const { auth } = require("../middlewares/middlewares");
 const app = express.Router();
 
+app.get("/me", auth, controllers.loadUser);
+app.post("/login", controllers.login);
 app.delete("/pets/:id", controllers.destroyPet);
```

Comment:
Add middleware to not public routes

Reply:
Added the "auth" middleware in not public routes